README.md

```diff
@@ -57,39 +57,54 @@ dependencies {
 
 The core of lite-for-jdbc is the Db class. A Db object is intended to be used as a singleton and
 injected as a dependency in repository classes. It requires a DataSource constructor argument,
-and there is a DataSourceFactory to help with that.
+and there is a DataSourceFactoryRegistry to help with that.
 The typical recommendation is to use Hikari, which is configured with reasonable defaults, but you can customize
 it to any DataSource.
 Examples:
 
-Hikari:
+Using DataSourceFactoryRegistry directly:
 
 ```kotlin
-val dataSource = DatasourceFactory(DbConfig(
-    username = "user",
-    password = "password",
-    databaseName = "dbName"
-)).dataSource()
+val config = DbConfig(
+  type = "H2_INMEM",
+  username = "user",
+  password = "password",
+  databaseName = "dbName"
+)
+val datasource = DataSourceFactoryRegistry.dataSource(config)
```

Comment:
Seems like from an external point of view it's still a `DataSourceFactory`. i.e. It's a factory creating a datasource for you.  Maybe the registry is an internal/private concept.?